dev/Dockerfile-ingestion

```diff
@@ -0,0 +1,7 @@
+FROM python:3.8
+
+RUN pip install boto3==1.18.38 requests==2.26.0 docker==5.0.2
```

Comment:
is there value to using a separately-defined environment (e.g. a poetry specification) here? Might I want to develop/test these scripts outside the docker environment, or is that too much hassle to support?

Reply:
Do we plan to support other languages than Python? Otherwise we could just use a venv/poetry here.